Cargo.toml

```diff
@@ -12,11 +12,11 @@ links = "tesseract"
 build = "build.rs"
 
 [dependencies]
-leptonica-sys = "~0.4"
+leptonica-sys = { path = "../leptonica-sys" }
 
 [build-dependencies]
 bindgen = "0.64"
+cmake = "0.1"
```

Comment:
Do you know what's standard with sys libraries and bundled code?

For example would it be normal to have a [crate feature](https://doc.rust-lang.org/cargo/reference/features.html) to give crate users an option to if they use the system library or build and use the bundled library?